binja_script.py

```diff
@@ -40,20 +40,27 @@ def get_widest_func(bv: binaryninja.BinaryView, addr: int) -> binaryninja.Functi
 
 
 # workaround for BinaryView.add_tag not supporting auto tags
-def add_auto_data_tag(bv: binaryninja.BinaryView, addr: int, name: str, desc: str) -> None:
-    tag = binaryninja.core.BNCreateTag(bv.get_tag_type(name).handle, desc)
-    binaryninja.core.BNAddTag(bv.handle, tag, False)
-    binaryninja.core.BNAddAutoDataTag(bv.handle, addr, tag)
+def add_data_tag(
```

Comment:
Is the comment above still valid btw?

Reply:
May be worth to detail the problem a bit more or why this is a workaround/how it fixes it etc

Reply:
Should still be valid, as the PC tag still uses auto tags.